scripts/zones/West_Sarutabaruta/mobs/Numbing_Norman.lua

```diff
@@ -28,10 +28,10 @@ function onAdditionalEffect(mob,target,damage)
 end;
 
 -----------------------------------
--- onMobDeath
+-- onMobDespawn
 -----------------------------------
 
-function onMobDeath(mob, player, isKiller)
+function onMobDespawn(mob)
     checkRegime(player,mob,61,2);
```

Comment:
@teschnei 

scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua:function onMobDeath(mob, player, isKiller)
scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua-    local AV_CHANCE = 25;
scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua-    if (AV_CHANCE > math.random(0,99)) then
scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua-        SpawnMob(16912876, 180);
scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua-    end
scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua-end;

or in Rock_Lizard.lua under onMobDeath:
...  
          if (math.random(1,20) == 5) then
...

would be rerolled for each party member would'nt it?